src/instruments/view/instrumentspaneltreemodel.cpp

```diff
@@ -219,11 +232,13 @@ bool InstrumentsPanelTreeModel::removeRows(int row, int count, const QModelIndex
         parentItem = m_rootItem;
     }
 
+    m_isLoadingBlocked = true;
```

Comment:
Please explain the reason for this flag. InstrumentsPanelTreeModel::load() should be called before any functions in this model

Reply:
Problem here is `InstrumentsPanelTreeModel::load()` will be called everything parts are changed, so also when parts are removed or moved.
This flag prevents the model to be reset while rows are moved/removed.